src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,4 +1,12 @@
 package frc.robot;
 
 public class RobotMap {
+    public interface CAN {
```

Comment:
blank line at the start of the interface

Reply:
ditto

Reply:
add please

Reply:
> ditto

grammar*